web/src/app/webview/src/keymanEngine.ts

```diff
@@ -69,7 +69,6 @@ export default class KeymanEngine extends KeymanEngineBase<WebviewConfiguration,
       pathConfig: this.config.paths,
       // When hosted in a WebView, we never hide the Web OSK without hiding the hosting WebView.
       activator: new StaticActivator(),
-      embeddedGestureConfig: buildEmbeddedGestureConfig(this.config.softDevice),
```

Comment:
- [ ] Undo this change.

Your changes in `oskConfiguration.ts`, which remove the Android-specific parts of the function being called here, should be sufficient.

This change _additionally_ removes the globe-key hint message data that allows the iOS app to provide a hint about the keyboard-change key.  I'm not sure if Android has that as well; if it does, Android also uses the same thing.  (See lines 43-47 of `oskConfiguration.ts`.)

While not the exact matching commit, this is the corresponding globe-hint section to be restored:

https://github.com/keymanapp/keyman/blob/e7dedfc8ee46eba90ff3c725c4987eba8d8f1b32/web/src/app/webview/src/oskConfiguration.ts#L42-L46